core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionFixture.scala

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.dsde.rawls.expressions
+
+// test data for Expression Validation and Parsing
+
+trait ExpressionFixture {
+  // Inputs which will not validate, so they don't need to be handled by the parser
+  // ""
+
+  val parseableInputExpressions = Seq(
+    "this.gvcf",
+    "workspace.gvcf",
+    "workspace.library:cohort",
+    "this.library:cohort",
+    "this.arbitrary:whatever",
+    "workspace.arbitrary:whatever",
+    "this.case_sample.foo:ref.bar:attribute",
+    "workspace.yes.we.can",
```

Comment:
(for instance, i was about to say "wait no, workspace expressions can only evaluate direct attributes, not chains through other entities"... but then i looked up the parser and turns out i was wrong!)